conformance/utils/suite/suite.go

```diff
@@ -55,6 +56,10 @@ type Options struct {
 	Debug           bool
 	BaseManifests   string
 	NamespaceLabels map[string]string
+	// HostNetworkPortRangeStart allows using a custom port range for host-networked pods.
+	// This is useful to avoid port conflicts on different clusters.
+	// The amount of hostNetwork pods required may change, check base/manifests.yaml {{ .Ports }} indices.
```

Comment:
Is this the "official" documentation for that parameter? If so, mention the default value?
Also, it would probably be good to document exactly how many values are needed (and note elsewhere that if we expand the range that you need to update the value here).